modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -203,8 +203,8 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
     for {
       stepType  <- calcStepType(config)
       inst      <- toInstrumentSys(stepType.instrument)
-      systems   <- calcSystems(config, stepType)
-      headers   <- calcHeaders(config, stepType)
+      systems   <- calcSystems(config, stepType, inst)
```

Comment:
I realized I could simplify the code a bit passing `InstrumentSystem[F]` rather than reading it everywhere

Reply:
:+1: